doc/code.html

```diff
@@ -202,25 +202,35 @@ <h3 id="ImportPaths">Import paths</h3>
 </p>
 
 <p>
-We'll use <code>github.com/user</code> as our base path. Create a directory
-inside your workspace in which to keep source code:
+We'll use <code>github.com/user</code> as our base path for our Go programs. Create a new 
```

Comment:
We should stop using github.com/user as the base path for our program with modules. Instead, they should just be able to do something like create a folder on their desktop or in Documents or something. Also this looks like a windows-specific path

Reply:
@mdwhatcott I wasn't sure whether to write this from a Go 1.12 point of view, in which setting the environment variable will be redundant (I think?). I'll update this though to reflect current release and ensure it's easily delete-able. 

@deanveloper If the `go mod init` command can't automatically determine the module path from the likes of VCS meta-data then you need to supply the module path. I felt keeping github.com/user was consistent with the rest of the content already there and would minimize confusion if the user hadn't already run `git init` + `git remote add origin ...`.

To both - I can add a line that explicitly states that the new project directory be created outside of the `GOPATH` and within a directory the user is comfortable with?

Reply:
Sounds good @elliotforbes

Reply:
@mdwhatcott + @deanveloper I've updated the page to include both your suggestions. Have a look and let me know what you think :)